tfjs-backend-webgpu/src/buffer_manager.ts

```diff
@@ -26,8 +26,10 @@ export class BufferManager {
 
   constructor(private device: GPUDevice) {}
 
+  // The upload buffer is not reusable.
   acquireUploadBuffer(size: number, usage: GPUBufferUsageFlags) {
```

Comment:
We can remove this and call acquireBuffer directly.

Reply:
`acquireBuffer`/`releaseBuffer` does many other things to manage `freeBuffers` and `usedBuffers` for reusing. They are all not needed for staging buffers. So I think the current way is more clear and efficient.

Reply:
Let's keep it as is and do changes in a later PR if needed.